src/server/src/modules/facultyMember.ts

```diff
@@ -124,9 +125,18 @@ export const getPostedJobs = async (
  * @description Get a list of students who applied to a job.
  * @param {number} facultyMemberId - Id of faculty member
  * @param {number} jobId - Id of the job
+ * @param {number[]} departmentIds - List of department ids. [-1] if not specified.
+ * @param {ClassStanding[]} classStandings - List of preferred class standings.
+ * @param {number} minimumGpa - minimum GPA.
  * @returns Promise
  */
-export const getApplicants = async (facultyMemberId: number, jobId: number) => {
+export const getApplicants = async (
+    facultyMemberId: number,
+    jobId: number,
+    departmentIds: Student['departmentId'][],
+    classStandings: Student['classStanding'][],
+    minimumGpa: number,
```

Comment:
Student['gpa']

Reply:
Student['gpa'] has type "number | undefined". Since minimumGpa is a required field, we should use "number" here.

Reply:
sounds good